### PR TITLE
Add the ability to set the tarLongFileMode of the assemblies

### PR DESCRIPTION
**Pull request by cunningt**
Add the ability to set the tarLongFileMode of the assemblies, keep the default of gnu

### CODE REVIEW

**Comment by EricWittmann** [comment]
@jpechane I assume you have no concerns about this?

**Comment by cunningt** [comment]
Please hold off on merging - I think I'm going to need to fix something - getting an error on this in our dev build.

**Comment by jpechane** [comment]
@EricWittmann Definitely not a problem for me

File: distro/pom.xml [hunk 1]
```diff
@@ -19,6 +19,15 @@
         <module>connect-converter</module>
     </modules>
     <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-assembly-plugin</artifactId>
+                    <version>3.1.0</version>
```
Comment (alesj) [inline distro/pom.xml]: Versions should go into top pom.xml ...

**Pull request merged**